### ISSUE TITLE
Default linux path not working on Windows with $has_cluster

### ISSUE THREAD

**Issue by agoddard** [sensu/sensu-puppet#790]
```
newparam(:base_path) do 
  desc 'The base path to the client config file' 
  defaultto '/etc/sensu/conf.d/' 
end
```

This doesn't work on windows when variable $has_cluster in manifests/rabbitmq/config.pp is true. I get this error: 

```
Error: /Stage[main]/Sensu::Rabbitmq::Config/Sensu_rabbitmq_config[hostname.fqdn]: Could not evaluate: No such file or directory @ rb_sysopen - \etc\sensu\conf.d\rabbitmq.json 
```

**Comment by ghoneycutt** [comment]
@alvagante this is high priority, could you please look into it?

**Comment by alvagante** [comment]
@ghoneycutt sorry, seen it just now. Going to work on it now.

**Comment by ghoneycutt** [comment]
Fixed in v2.33.0
